contribution-guidelines-engineering.md

```diff
@@ -0,0 +1,28 @@
+# Contribution Guidelines for Engineering
+
+This post will provide some contribution guidelines for CSS/Scss in Atlas.
+
+## When making a code change
+
+- be careful and purposeful.
+- test your code change locally using the site.
+- ask yourself, does this change need to be documented?
+  - if so, and the change is to our styles, please ensure that you make a documentation change in the same pull request
+  - if so, and the change is to our development environment, please make sure the appropriate readme file is updated/created.
+- fill out a **changeset** using the [changeset cli](https://github.com/atlassian/changesets/blob/main/docs/intro-to-using-changesets.md).
+  - if the change is to styles, you must fill out a changeset.
+  - if the change is to the site's infrastructure, you should fill out a changeset. _Note: we do not need semver for the site, but changesets will also provide a nicely organized changelog for improvements across the site._
+- make a pull request and whenever possible...
+  - include a work item number
+  - include a preview link to PR version of the site for easier functional reviewing of your code. (`preview-<pr-number>` or `<origin>/pulls/<pr-number>/path/to/page/`)
+- avoid making breaking changes, unless you really, really have to.
+- if you don't know what might constitute a breaking change ask a team member or start a discussion.
+- checkout the readme of the folder you're working in. Each folder should have a readme detailing advice on the purpose the folder and guidelines for contributing.
+
+## When writing new Scss
+
+- ensure signoff from the design systems team has been reach before you begin work
```

Comment:
"has been reached"?

Reply:
Would it make sense to notate the design systems team members here so it's known who to contact for signoff?

Reply:
I'm not to keen on maintain a list of personnel here, I think we should just leave it for now. Add a new sentence below, let me know what you think.

Reply:
I see that my first comment has been addressed. I still see only one sentence though. What's the new sentence? 
![image](https://user-images.githubusercontent.com/64044661/117349312-1b1e5c00-ae60-11eb-88c9-a4382ad8791e.png)